data/templates/centos-ks

```diff
@@ -17,8 +17,10 @@ selinux --permissive
 services --enabled=NetworkManager,sshd
 network --device=<%=macaddress%> --noipv6 --activate
 
-#enable syslog
-logging --host=<%=server%> --level=info
+# enable syslog
+<% if (remoteLogging) { %>
```

Comment:
Since remoteLogging is optional, if remoteLogging is not passed in parameter, your code will report unreference error. so Please copy the parameter `kvm` and change to `<% if (typeof remoteLogging !== 'undefined' && remoteLogging) { %>1.